### ISSUE TITLE
FP for oelint.vars.srcurifile

### ISSUE THREAD

**Issue by priv-kweihmann** [priv-kweihmann/oelint-adv#231]
recipe
```
A = "1"
inherit foo
```
foo.bbclass
```
SRC_URI += "https://some.corp.org/${PN}"
```
reports false positive `First item of SRC_URI should not be a file:// fetcher, if multiple fetcher are used`
